style.css

```diff
@@ -762,7 +767,162 @@ input[type=submit]:disabled {
   height: 100vh;
 }
 
+.modal {
+  width: 100%;
+  height: 180%;
+  background-color: rgba(0, 0, 0, 0);
+  backdrop-filter: blur(6px);
+  position: absolute;
+  top: 0;
+  display: none;
+  justify-content: center;
+  align-items: center;
+}
```

Comment:
Kindly update your popup window in the mobile version to match the figma design better🙏

Your design:
<img width="197" alt="image" src="https://github.com/YoutLone/Portfolio/assets/37116322/21f7bd00-3acc-4964-8548-92eba8c3507f">

Figma design:
<img width="126" alt="image" src="https://github.com/YoutLone/Portfolio/assets/37116322/ee024aaf-066f-415c-983c-d40d062f5f59">

A few things to note:
- Please update your popup window to start at the top of the page like the figma design
- Please make your popup window fullscreen and prevent user scrolling to stop an overlap of your design
- Your popup window only looks good on certain screen sizes in your mobile version. Please make sure your popup window uses responsive widths to make sure it is responsive on all mobile screen sizes (375px - 768px)